style.css

```diff
@@ -0,0 +1,361 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	width: 100%;
+	max-width: 5000px;
+	margin: 20x auto; 
+}
+
+.navbar {
+	display: flex;
+	justify-content: flex-start;
+	gap: 50px;
+	padding: 15px;
+	padding-left: 45px;
+	background-color: #9a7b5c;
+	font-family: 'Roboto Flex', sans-serif;
+	font-size: 20px;
+	letter-spacing: 3px;	
+}
+
+.contact-link {
+	text-decoration: none;
+	color: #dac7b2;
```

Comment:
this colour doesn't have enough contrast against the background for accessibility, either the text needs to be lighter or the background darker